install_dependencies.R

```diff
@@ -1,5 +1,9 @@
 install.packages(c('testthat', 'vdiffr', 'spelling','covr','remotes'), repos = 'http://cran.us.r-project.org', type='win.binary')
-install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')
+download.file('https://github.com/Open-Systems-Pharmacology/rSharp/releases/download/v1.0.0/rSharp-v1.0.0-Windows-r_4.4.0.zip', destfile = "rsharp.zip", mode = "wb")
+unzip("rsharp.zip")
+rSharp_archive <- list.files( pattern = "rSharp.*\\.zip")
+install.packages(rSharp_archive, repos = NULL, type = "binary")
```

Comment:
Because github built artifacts are zipped. So the package is one level deeper in the archive, and R doesn't expect that.

Reply:
But it won't be needed once we go to GHactions because it will be able to build from source directly (see https://github.com/Open-Systems-Pharmacology/OSPSuite-R/actions/runs/9204757821/job/25318948079#step:3:5051)